src/pages/MembersPage/MembersTable.tsx

```diff
@@ -512,7 +512,7 @@ const MemberRow = ({
                           address: user.address,
```

Comment:
`name: user.name`
missed updating this line

Reply:
done